libs/accounts/recovery-phone/src/lib/recovery-phone.service.spec.ts

```diff
@@ -52,12 +54,25 @@ describe('RecoveryPhoneService', () => {
       smsPumpingRiskThreshold: 75,
     },
   };
+  const mockGetFormattedMessage = async (code: string) => {
+    return {
+      msg: `${code} is your Mozilla verification code. Expires in 5 minutes.`,
```

Comment:
Maybe I'm over thinking but I wonder if we need to include the expiration. The expiration is on the webpage. For reference Twilio sends this message, `Your Twilio verification code is: 859440` and does not include expiration time. In this case, the text lenght would be rough 40 characters so the risk for translations being over 160 would be much lower.